sponsorship/index.md

```diff
@@ -97,7 +93,19 @@ Participants are allowed to share their information with you at your stall.
 Give them a good reason to do so. Make a compelling pitch for yourself
 or offer goodies in exchange for contact information.
 
+<hr>
 
 ## Contact
-
 - Mail: [sponsor@in.pycon.org](mailto:sponsor@in.pycon.org)
+- Abhishek Mishra, Sponsorship Coordinator
+  - +91 70664 87364
+- Chandan Kumar, Sponsorship Coordinator
+  - +91 96072 64082
+- Sayan Chowdhury, PyCon India 2020 Chair
+  - +91 96869 92532
```

Comment:
Are you sure you want to add phone numbers?

Reply:
Yes, We've been having phone numbers on the pages. So, let go ahead and keep it.

Reply:
Okay.